web/src/main/java/org/mskcc/cbio/oncokb/controller/advice/ApiHttpError.java

```diff
@@ -0,0 +1,142 @@
+package org.mskcc.cbio.oncokb.controller.advice;
+
+import org.springframework.http.HttpStatus;
+
+/**
+* Represents an HTTP error response with detailed information about the error.
+* This class captures the title, status, detail, path, and message associated with the HTTP error.
+*
+* Example JSON representation:
+* <pre>
+* {
+*   "title": "Method Not Allowed",
+*   "status": 405,
+*   "detail": "Request method 'POST' is not supported",
+*   "instance": "/index.html",
+*   "message": "error.http.405",
+*   "path": "/index.html"
+* }
+* </pre>
+*/
+public class ApiHttpError {
+    /** The title of the error, typically representing the HTTP status text. */
+    private String title;
+
+    /** The HTTP status code of the error (e.g., 404, 500). */
+    private int status;
+
+    /** A detailed message explaining the error. */
+    private String detail;
+
+    /** The request path that caused the error. */
+    private String path;
+
+    /** A message key that is standardize to help with client side error handling, such as "error.http.404". */
```

Comment:
Is this true? This is a message key and is standardized?

Reply:
It's my best guess as to why it exist. I could be bad a googling, but I couldn't find what it's supposed to mean. Honestly `detail` is the only field that needs to be there. All the other information is already in a http response.